ta4j-examples/pom.xml

```diff
@@ -17,6 +17,12 @@
             <artifactId>ta4j-core</artifactId>
             <version>0.12-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Dependency not needed

Reply:
It is if you want Travis CI on the examples.

Reply:
Why would we need Travis CI in the examples?

Reply:
So we wouid know when the examples break, as in this case.  CI would have caught these problems before I did.  I understand if you don't want CI on the examples, but somebody should re-test them from time to time.

Reply:
Would be nice to test the examples automatically, but it could be very confusing for users if they are confrontated with a test framework in the quickstarts

Reply:
But they would never see the test framework unless they looked at `QuickstartTest` or `pom.xml` in `ta4j-test`.  It's still just a stand-alone application, no different from the other examples.